content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -4,5 +4,10 @@ title: What´s new
 layout: bundle
 ---
 
-
 Release 10.15.0 includes the following new features or major feature enhancements.
+
+### New All devices view
```

Comment:
In the user documentation, we talk about "pages" and "tabs", the term "view" is rather technical. So it should be "All devices page" here. Also, I would not use the term "new" as the page is not new, better say "Modified" or "Improved".